src/ui/hooks/recordings.ts

```diff
@@ -45,7 +48,8 @@ export function useGetRecording(recordingId: RecordingId) {
   const recording = data?.recordings[0];
   // Tests don't have an associated user so we just let it bypass the check here.
   const isAuthorized = isTest() || recording;
-  return { recording, isAuthorized, loading };
+  const workspaceName: string | null = data?.workspace?.name;
```

Comment:
This is in the wrong place given the query you are doing, it would be `recording?.workspace?.name`

Reply:
argh thanks!